<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="judul">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;MyOwnStreamCipher&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="jenis_pesan">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Jenis Pesan</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input_key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="metode">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Metode</string>
    </property>
   </widget>
   <widget class="QLabel" name="key">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QLabel" name="judul_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plainteks</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 197, 15);</string>
    </property>
    <property name="text">
     <string>Browse File</string>
    </property>
   </widget>
   <widget class="QLabel" name="browse_filename">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>395</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="judul_output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cipherteks</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="save_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 197, 15);</string>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="save_filename">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>550</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="button">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>590</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Poppins</family>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(249, 197, 15);</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="jenis_file">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>6</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>File</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="jenis_plainteks">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>6</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Plainteks</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>100</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QRadioButton" name="metode_encrypt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>6</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="metode_decrypt">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>6</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Poppins</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
